.dockerignore

```
@@ -0,0 +1,5 @@
+.git
+build
+!build/docker/config.yaml
```

This should be `deploy`, right?
```suggestion
deploy
!deploy/docker/config.yaml
```